pkg/configtx/consortiums.go

```diff
@@ -244,3 +244,21 @@ func implicitMetaAnyPolicy(policyName string) (*standardConfigPolicy, error) {
 		value: implicitMetaPolicy,
 	}, nil
 }
+
+// getConsortiumOrg returns the organization config group for a consortium org in the
+// provided config. It returns an error if a named consortium or a consortium org was not found
+// in the config with the specified name.
+func getConsortiumOrg(config *cb.Config, consortiumName string, orgName string) (*cb.ConfigGroup, error) {
+	consortiumsGroup := config.ChannelGroup.Groups[ConsortiumsGroupKey].Groups
+	consortiumGroup, ok := consortiumsGroup[consortiumName]
+	if !ok {
+		return nil, fmt.Errorf("consortium %s does not exist in channel config", consortiumName)
+	}
```

Comment:
This check is no longer needed per the point above (as well as the returned error).